### DISCUSSION TITLE
Bad measurement in yaw angle.

### DISCUSSION THREAD

**Discussion by Wallat1** [bitcraze/discussions#1557] [Q&A]
Hi.

I have the following problem, when I put the crazyflie with the following attitude roll = 90, pitch = 0 and yaw = 90, in the cfclient the yaw angle measures 110 or 150 degrees, do you have any idea why this happens? If so, could you please tell me how to solve the problem?.

I would appreciate your help.

Regards.

**Comment by gemenerik** [comment]
What do you mean with "put"?
Yaw is relative to how it was initialized, so if I turn on my CF, turn it 90 degrees, it reads 90 degrees as expected.

**Reply by Wallat1** [reply]
Thank ypu for answering, I mean "to position", when I position the crazyflie in roll = 90, pitch = 0 and yaw = 90.

To clarify, first I turn it on and wait for the measurement to be approximately 0 at each angle, then I position it with the aforementioned attitude.  After the crazyflie starts sending bad information.

I would appreciate your help.

Regards.

**Reply by gemenerik** [reply]
Are you using any expansion decks? Any custom firmware or latest release with default settings?
